ddp-workspace/projects/ddp-mbc/src/app/components/partners/partners.component.ts

```diff
@@ -0,0 +1,174 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-partners',
+  template: `
+  <div class="row">
+    <div class="col-lg-3 col-md-3 col-sm-3 col-xs-12 Message-partners">
+        <a href="http://www.mbcn.org/" target="_blank"><img lazy-resource class="partner-logo"
+                src="./assets/images/logo-mbcn.png" [alt]="'Partners.MBCN' | translate"></a>
+    </div>
+    <div class="col-lg-3 col-md-3 col-sm-3 col-xs-12 Message-partners">
+        <img lazy-resource class="partner-logo" src="./assets/images/logo-avon.svg"
+            [alt]="'Partners.Avon' | translate">
+    </div>
+    <div class="col-lg-3 col-md-3 col-sm-3 col-xs-12 Message-partners">
+        <a href="https://www.mbcalliance.org/" target="_blank"><img lazy-resource class="partner-logo"
+                src="./assets/images/logo-mbca.png" [alt]="'Partners.Alliance' | translate"></a>
+    </div>
+    <div class="col-lg-3 col-md-3 col-sm-3 col-xs-12 Message-partners">
+        <a href="http://www.lbbc.org/" target="_blank"><img lazy-resource class="partner-logo"
+                src="./assets/images/logo-lbbc.png" [alt]="'Partners.LBBC' | translate"></a>
```

Comment:
One real change is here.
I decided to extract this section in the separate component due to size and also now we need to make changes only in the one place if we want to update something.